core/modules/system/system.admin.inc

```diff
@@ -1610,19 +1588,30 @@ function system_run_cron_submit($form, &$form_state) {
  * @see system_logging_settings_validate()
  */
 function system_logging_settings($form, &$form_state) {
+  $site_config = config('system.core');
   $form['#config'] = 'system.core';
-  $form['error_level'] = array(
+
+  $form['error_display'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Errors displayed'),
```

Comment:
Perhaps change this to "Error messages to display" and remove the title from the radios contained in this fieldset?

Reply:
I don't think it's possible to have a form element without a title (?) but even if it were, it would be bad for accessibility. Can you think of another fieldset label that might be better, or should I still try this suggestion?

Reply:
🤔 ...I actually felt the same way, and was to do some research re what would be an acceptable solution (I was in need to do the same for the 404/403 path fields in the PR I've filed against yours) ...but then got distracted and never did that.

Reply:
...oh joy! ...turns out you can hide element titles in a way that still retains accessibility, by using `'#title_display' => 'invisible'` 😀 ...I've seen that in the past somewhere; just never had the need for it till now.